README.md

```diff
@@ -53,6 +53,15 @@ end
 Everypolitician::Index.new.all_legislatures.each do |legislature|
   puts "#{legislature.name} in #{legislature.country.name} has #{legislature.person_count} member(s)"
 end
+
+# Fetch Everypolitician::Popolo::LegislativePeriod for the latest term
+# Note that this may involve a **substantial download**
+assembly = Everypolitician.country(slug: 'Albania').legislature(slug: 'Assembly')
+term_details = assembly.latest_term
+puts 'Members of the Assembly:'
+term_details.memberships.each do |m|
+  puts m.person.name
+end
```

Comment:
I'm not convinced by this example, as it implies that `person` will be unique there. If we want to show a list of people, rather than memberships, I think we should make them unique first.